tests/profiles/googlefonts_test.py

```diff
@@ -2358,37 +2356,37 @@ def test_check_name_subfamilyname():
 
       if name.nameID == NameID.FONT_SUBFAMILY_NAME:
           ttFont['name'].names[i].string = value.encode(name.getEncoding())
-    print (f"Test PASS with filename='{filename}', value='{value}', style_with_spaces='{style_with_spaces(filename)}'...")
+    style = expected_style(ttFont)
+    print (f"Test PASS with filename='{filename}', value='{value}', "
+            "style_win='{style.win_style_name}', "
+            "style_mac='{style.mac_style_name}'...")
```

Comment:
you will need to place an 'f' before each line which references variables. Add those here, please. Travis will fail here.